RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -87,17 +86,6 @@ public CodeExplorerItemViewModel SelectedItem
                 _selectedItem = value;
                 OnPropertyChanged();
 
-                if (_selectedItem is CodeExplorerProjectViewModel)
-                {
-                    var vbe = _selectedItem.GetSelectedDeclaration().Project.VBE;
-                    var project = vbe.VBProjects.Cast<VBProject>().FirstOrDefault(f => f.HelpFile == _selectedItem.GetSelectedDeclaration().Project.HelpFile);
-
-                    if (project != null)
-                    {
-                        vbe.ActiveVBProject = project;
-                    }
-                }
-
```

Comment:
I removed that for a couple reasons.

A) I was only setting that when the project was selected in the VBE.  Nothing else was being synced, and it wasn't even doing what we intended.
B) It was created a QualifiedModuleName somewhere, causing a slight flicker as the VBE quickly swapped the active project.  Actually, I need to make that a property so we do this as little as possible.